src/app/js/fields/selectors.js

```diff
@@ -121,16 +120,19 @@ const getAllListFields = createSelector(getCollectionFields, fields =>
     fields.filter(f => !f.composedOf),
 );
 
-const getOverViewCol = type => fields => {
-    const result = fields.filter(f => f.overview === type);
+const findFieldWithOverviewID = id => fields => {
+    const result = fields.filter(f => f.overview === id);
```

Comment:
Why use `filter`, while there is also `find`.
http://devdocs.io/javascript/global_objects/array/find

Reply:
just now, I try to use find and I don't no why but it's can not works : 

```
const findFieldWithOverviewID = id => fields =>
    fields.find(f => f.overview === id);
```

so I prefer to leave as it is

Reply:
But that's a `name`, that should be returned, not a field. (See line 125 below).